src/main/java/com/example/fooddelivery/service/UserService.java

```diff
@@ -0,0 +1,27 @@
+package com.example.fooddelivery.service;
+
+import com.example.fooddelivery.model.User;
+import com.example.fooddelivery.repository.UserRepository;
+
+/**
+ * UserService for user operations.
+ */
+public interface UserService {
+
+    /**
+     * Get user from DB
+     *
+     * @param email user email
+     * @param password user password
+     * @return {@link User} entity
+     */
+    User getUserByEmailAndPassword(String email, String password);
+
+    /**
+     * Save user via {@link UserRepository}
+     *
+     * @param user entity
+     * @return saved entity
+     */
+    User save(User user);
```

Comment:
This looks a little bit weird because you don't do any other logic just calling repository.
Why you don't call Repository `save` method directly from the facade? Because you are already in the service layer.
Please google it, maybe I'm wrong.

Reply:
As we discussed with more experienced devs we should keep both service and facade even if it only returns a call of the other one with same parameters